client/src/Component/SMSModal.js

```diff
@@ -1,23 +1,54 @@
-/* eslint-disable jsx-a11y/label-has-for */
+
 import { useState } from "react";
 import Modal from "react-bootstrap/Modal";
 import Button from "react-bootstrap/Button";
 
+
+
+
 function SMSModal(props) {
     const [show, setShow] = useState(false);
     const hideModal = () => setShow(false);
     const showModal = () => setShow(true);
 
+    let localMessage = localStorage.getItem("message");
+    if (localMessage === null) {
+      localMessage = "Hi, this is Bikes For Refugees. Bikes are now available for you to pick up. Please click LINK to choose a pick up date. For any questions, please contact 07900000000.";
+    }
+    const [message, setMessage] = useState(localMessage);
+
+    const handleChange = (e) => {
+      localStorage.setItem("message", e.target.value);
```

Comment:
Setting localStorage is considered a "side effect" because we're doing something that's outside of React. In React, we should use [`useEffect` to perform side effects](https://reactjs.org/docs/hooks-effect.html), and we can tell it to update localStorage whenever the `message` state is updated with something like the following which should be outside the `handleChange` function:

```js
useEffect(() => {
  localStorage.setItem("message", message);
}, [message]);
```